tool/generate_code_coverage.sh

```diff
@@ -6,4 +6,4 @@ set -o pipefail
 dart pub global activate coverage
 
 # Run tests and generate lcov
-dart test --test-randomize-ordering-seed=random --coverage=coverage && dart pub global run coverage:format_coverage --lcov --in=coverage --out=coverage/lcov.info --packages=.packages --report-on=lib
```

Comment:
is the random ordering the default now? or do we know we don't want that for coverage or something?

Reply:
i have it on good authority that randomness is good for us https://www.betterment.com/engineering/stability-through-randomness

Reply:
That was a total copy pasta mistake (got it from `test_track_dart_client` 🤦 ). Will fix.